workflow/scripts/metrics.py

```diff
@@ -79,6 +86,7 @@ def PlotQuantilePlot(EnhancerList, title, outdir):
     fig = ax.get_figure()
     outfile = os.path.join(outdir, i+str(title)+".pdf")
     fig.savefig(outfile, format='pdf')
+    plt.clf()
```

Comment:
Need to clear the plots so that their attributes don't transfer to the next plots